### PR TITLE
add: epic games company profile to README.md

### PR DESCRIPTION
**Pull request by rakhadjo**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by auto-nylen** [comment]
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

**Comment by rakhadjo** [comment]
@dougaitken

**Comment by dougaitken** [comment]
Hey @rakhadjo please ONLY push the Epic Games addition -leave out the sections of the readme with the alphabet on this one.

**Comment by rakhadjo** [comment]
@dougaitken done, reverted my master to two commits behind. Personally thought adding the index feature would've been pretty cool, thanks for the input, Doug, all the best 👍

**Comment by dougaitken** [comment]
Of course @rakhadjo it is a great addition but I'd like that to be a branch of its own please so we can discuss in a PR purely for that.

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ Name | Website | Region
 [Enok](/company-profiles/enok.md) | https://www.enok.co/ | USA
 [Envato](/company-profiles/envato.md) | https://envato.com/ | Worldwide
 [Epam](/company-profiles/epam.md) | https://epam.com/ | Worldwide
+[Epic Games](/company-profiles/epic-games.md) | https://www.epicgames.com/site/en-US/careers | USA & Worldwide
```
Comment (dougaitken) [inline README.md:160]: Remove USA if it is worldwide

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by dougaitken** [comment]
One small edit @rakhadjo

**Review by rakhadjo — COMMENTED** [review]
USA deleted from `README.md` profile entry

**Review by dougaitken — APPROVED** [review]

**Pull request merged**